index.bs

```diff
@@ -10765,7 +10765,7 @@ registerProcessor('vumeter', class extends AudioWorkletProcessor {
 import VUMeterNode from './vumeter-node.js';
 
 const context = new AudioContext();
-context.audioWorklet.addModule('vumeter-processor.js').then(() =&lt; {
+context.audioWorklet.addModule('vumeter-processor.js').then(() =&gt; {
```

Comment:
We should probably change all of our code snippets to use `<xmp>` instead of `<pre>` so we don't have to do this kind of escaping.

But that's not your responsbility. :-)

Reply:
#2100 is a copy of this PR with those changes (only for the AudioWorkletNode examples) as well 😄

Reply:
Let's merge #2100 as well. Thanks!